packages/mobile/src/components/details-tile/DetailsTile.tsx

```diff
@@ -273,6 +318,25 @@ export const DetailsTile = ({
     innerImageElement
   )
 
+  const playbackPositionInfo = useSelector((state) =>
+    getTrackPosition(state, { trackId })
+  )
+
+  const playText =
+    isNewPodcastControlsEnabled && playbackPositionInfo?.status
+      ? playbackPositionInfo?.status === 'IN_PROGRESS'
+        ? messages.resume
+        : messages.replay
+      : messages.play
+
+  const PlayIcon = (iconProps) =>
```

Comment:
ah so slightly unperformant to create a component here each time, i think maybe you can do 
```
const PlayIcon = condition ? IconRepeat : IconPlay
```
and pass that along?